SPECS/xcb-proto/xcb-proto.spec

```diff
@@ -2,20 +2,16 @@
 %define debug_package %{nil}
 Summary:        XCB protocol descriptions
 Name:           xcb-proto
-Version:        1.13
-Release:        16%{?dist}
+Version:        1.15
```

Comment:
There seems to be a `1.16` release as well

Reply:
Sure @mfrw  I will try it, but I could only find 1.15 as latest here: https://xcb.freedesktop.org/dist/ 
Do you happen to know any other source of xcb packages where I can get that 1.16 version?

Reply:
Yes, https://xorg.freedesktop.org/archive/individual/proto/ 
has 1.16